docs/tokens.md

```diff
@@ -1,25 +1,21 @@
 # Mintlayer Tokens
-**TODO Do we want Rust code in this doc?**
 
-Each transaction output must carry a data field. This field describes the purpose of the transaction.
+Each transaction output may carry a data field describing the purpose of the transaction.
 
-We must highlight the following at the moment:
-
-- Transfer Tokens or NFT
+The data field 
+- Transfer of Tokens or NFT
 - Token Issuance
 - Burning tokens
 - NFT creation
 
 All transactions must be signed on the client side. This means we cannot sign transactions on the node side.
-**TODO what is the connection between these two sentences?**
-In the future, there will be changes to the structures of transactions and we will be required to track them.
-**TODO by structures above to we mean rust structs? Why is this interesting to the user?**
 
+**TODO Add explanation of MLS-01**
 ## Transfer Tokens 
 
 **TODO sentence fragment, I don't understand...**
 **TODO When do we NOT use the TxData field?**
-For transfering funds to another person in a given UTXO. To send MLT we will use the MLT token ID, which is equal to 0. If the token ID is equal to the ID of the MLS-01 (**TODO what is MLS-01**) token, then the amount of token is transferred to the recipient. The commission is taken only in MLT (**TODO what is this commission**. If the token ID is equal to the ID of any NFT, then the data of this NFT is transferred to the recipient without changing the creator field. The UTXO model itself allows to determine the owner of the NFT.
+For transfering funds to another person in a given UTXO. To send MLT we will use the MLT token ID, which is equal to 0. If the token ID is equal to the ID of an MLS-01 token, then the amount of the token is transferred to the recipient. The commission is taken only in MLT (**TODO what is this commission**. If the token ID is equal to the ID of any NFT, then the data of this NFT is transferred to the recipient without changing the creator field. The UTXO model itself allows to determine the owner of the NFT **TODO Explain this**.
```

Comment:
What do you mean by commission? Do you mean the transfer fee? As it stands we enforce a MLT tx fee as we have no way of helping a block producer select the most valuable tx fee when they're in random tokens. Is a bentoken worth more than a roytoken? Are they both worth less than MLT?